text/0045-add-notion-of-protocol.md

```diff
@@ -0,0 +1,116 @@
+# Summary
+- Add notion of service protocol in libobs and OBS Studio
```

Comment:
I wonder if either of these would be clearer:
* Define supported protocols for services in libobs and OBS Studio
* Teach libobs and OBS Studio about supported protocols for services

Or something like that.

Reply:
I made some changes.